src/styles/colour/default.njk

```diff
@@ -0,0 +1,69 @@
+---
+layout: layout-example.njk
+---
+
+<h2 class="govuk-heading-m">Main colours</h2>
+
+{% for groupName, group in colours.Main %}
+
+  <h3 class="govuk-heading-s">{{groupName}}</h3>
+
+  <table class="govuk-body app-c-colour-list">
```

Comment:
I think if we're going to display this data as a table, we should add column headers.

Reply:
you're right, you need them for accessibility I think. However I don't know what to do when the table collapses for mobile view